.github/workflows/interop.yml

```diff
@@ -0,0 +1,52 @@
+name: Interoperability Testing
+on:
+  pull_request:
+  push:
+    branches:
+      - unstable
+
+jobs:
+  run-multidim-interop:
+    name: Run multidimensional interoperability tests
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/checkout@v3
+        with:
+          #repository: libp2p/test-plans
+          repository: status-im/libp2p-test-plans
+          path: test-plans
+          ref: nimsupport
+          submodules: true
+          fetch-depth: 0
+
+      - name: Build image
+        run: >
+          cd test-plans/multidim-interop/nim/v1.0 &&
+          make commitSha=$GITHUB_SHA image_name=nim-libp2p-head
```

Comment:
Cool! is this the commit sha for the current PR?

Reply:
Yes :)
https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables